components/code-editor/index.js

```diff
@@ -62,8 +140,19 @@ class CodeEditor extends Component {
 	}
 
 	render() {
-		return <textarea ref={ ref => this.textarea = ref } value={ this.props.value } />;
+		return <textarea ref={ ref => ( this.textarea = ref ) } value={ this.props.value } />;
```

Comment:
In other places I often see something like this:

```js
// constructor
this.bindTextarea = ref => ( this.textarea = ref );

// render
return <textarea ref={ this.bindTextarea } value={ this.props.value } />;
```

Reply:
Yeah, isn't this pattern preferred because it means `bindTextarea` doesn't change from `render`-to-`render` and so it is slightly more performant?